src/signals/incident-management/containers/Dashboard/components/Filter/styled.tsx

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import type { RefObject } from 'react'
+
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import styled, { css } from 'styled-components'
+
+import Refresh from '../../../../../../images/icon-refresh.svg'
+
+export const FilterContainer = styled.div<{
+  ref: RefObject<HTMLDivElement>
+}>`
+  width: 100%;
+  position: relative;
+`
+
+export const SelectContainer = styled.div`
+  display: flex;
+  font-size: 1rem;
+  height: ${themeSpacing(14)};
```

Comment:
We usually do height and with in pixels or percentages. Applies to all heights and widths in this file.

Reply:
I don't think anything changes right?